service/controller/resource/deployment/worker_deployment.go

```diff
@@ -42,6 +42,12 @@ func newWorkerDeployments(customResource v1alpha1.KVMConfig, release *releasev1a
 			return nil, microerror.Maskf(invalidConfigError, "error creating memory quantity: %s", err)
 		}
 
+		for _, hostVolume := range capabilities.HostVolumes {
+			if hostVolume.MountTag == "" || hostVolume.HostPath == "" {
```

Comment:
Should we push this validation into the CRD schema to make sure an invalid CR can't even be created? Could use a regex or set `MinLength` to 1.

Reply:
yes. it would make sense to move somewhere else. I agree here.

Reply:
@tfussell where should I move this? which repo?

Reply:
sure.

Reply:
done